analyzer/callable-index/src/test/java/eu/fasten/analyzer/callableindex/CallableIndexServerPluginTest.java

```diff
@@ -44,37 +48,35 @@ public void setUp() {
     @Test
     public void saveToDatabaseTest() throws IOException, RocksDBException {
         var rocksDao = Mockito.mock(RocksDao.class);
-        var json = new JSONObject("{\"payload\": {" +
-                "\"index\": 1," +
-                "\"product\": \"test\"," +
-                "\"version\": \"0.0.1\"," +
-                "\"nodes\": [1, 2, 3]," +
-                "\"numInternalNodes\": 2," +
-                "\"edges\": [[1, 2], [2, 3]]" +
-                "}}");
-        var graph = GidGraph.getGraph(json.getJSONObject("payload"));
+        var json = new JSONObject("{\"payload\": {}}");
+
+        var getTestResource = Objects.requireNonNull(Thread.currentThread().getContextClassLoader().getResource("gid_graph_test.json"));
+        json.getJSONObject("payload").put("dir", getTestResource.getPath());
+        JSONTokener tokener = new JSONTokener(new FileReader(getTestResource.getFile()));
+        var graph = GidGraph.getGraph(new JSONObject(tokener));
         callableIndexFastenPlugin.setRocksDao(rocksDao);
         callableIndexFastenPlugin.consume(json.toString());
         Mockito.verify(rocksDao).saveToRocksDb(graph);
     }
 
     @Test
-    public void consumeTest() {
-        var json ="{\"payload\": {" +
-                "\"index\": 1," +
-                "\"product\": \"test\"," +
-                "\"version\": \"0.0.1\"," +
-                "\"nodes\": [0, 1, 2]," +
-                "\"numInternalNodes\": 2," +
-                "\"edges\": [[0, 1], [1, 2]]" +
-                "}}";
-        callableIndexFastenPlugin.consume(json);
+    public void consumeTest() throws FileNotFoundException {
+        var json = new JSONObject("{\"payload\": {}}");
+
+        var getTestResource = Objects.requireNonNull(Thread.currentThread().getContextClassLoader().getResource("gid_graph_test.json"));
+        json.getJSONObject("payload").put("dir", getTestResource.getPath());
+        callableIndexFastenPlugin.consume(json.toString());
         assertNull(callableIndexFastenPlugin.getPluginError());
     }
 
     @Test
     public void consumeJsonErrorTest() {
-        callableIndexFastenPlugin.consume("{\"payload\":{\"foo\":\"bar\"}}");
+        var json = new JSONObject("{\"payload\": {}}");
+
+        var getTestResource = Objects.requireNonNull(Thread.currentThread().getContextClassLoader().getResource("gid_graph_test_err.json"));
```

Comment:
See comment above. This needs to be fixed throughout the class.

Reply:
Done!